cmd/sync.go

```diff
@@ -108,6 +109,25 @@ func (raw *rawSyncCmdArgs) cook() (cookedSyncCmdArgs, error) {
 		return cooked, fmt.Errorf("source '%s' / destination '%s' combination '%s' not supported for sync command ", raw.src, raw.dst, cooked.fromTo)
 	}
 
+	// Do this check seperately so we don't end up with a bunch of code duplication when new src/dsts are added
+	if cooked.fromTo.From() == common.ELocation.Local() {
+		result, err := filepath.Abs(cooked.source)
+
+		if err != nil {
+			return cooked, fmt.Errorf("failed to resolve absolute path from %s: %s", result, err)
+		}
```

Comment:
ToExtendedPath already does the filepath.Abs call

Reply:
True. I don't know entirely why I was worrying about that here.